torch/csrc/jit/fuser/executor.cpp

```diff
@@ -317,18 +317,8 @@ void launchFusion(
   fusion.launch_raw(numel, arguments);
 }
 
-bool runFusion(const int64_t key, Stack& stack) {
-  // Short-circuits if fusion isn't enabled
-  if (!canFuseOnCPU() && !canFuseOnGPU())
-    return false;
-
-  // Acquires the FusionSpec
-  auto maybe_spec = retrieve(key);
-  AT_ASSERT(maybe_spec);
-  auto& spec = *(*maybe_spec);
-
-  // Acquires inputs from stack
-  auto all_inputs = last(stack, spec.nInputs());
+using KernelInfo = std::tuple<std::shared_ptr<FusedKernel>, std::vector<at::Tensor>, at::Device>;
```

Comment:
I don't see how  this change relates to this PR. I also do not know what this stuff is. Please make a struct and document.

Reply:
This is github's diff utility being terrible. I pulled some stuff out of `runFusion`